<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieInfoDialog</class>
 <widget class="QDialog" name="MovieInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Movie Title</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yearLabel">
       <property name="text">
        <string>Year released: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="genreLabel">
       <property name="text">
        <string>Genres:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="runtimeLabel">
       <property name="text">
        <string>Runtime:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="actorLabel">
       <property name="text">
        <string>Starring:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="directorLabel">
       <property name="text">
        <string>Directed by:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="writerLabel">
       <property name="text">
        <string>Written by:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="watchButton">
     <property name="text">
      <string>Add to watch list</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
